instrumentation/ratpack/ratpack-1.7/library/src/main/java/io/opentelemetry/instrumentation/ratpack/v1_7/OpenTelemetryExecInterceptor.java

```diff
@@ -11,9 +11,9 @@
 import ratpack.exec.Execution;
 import ratpack.func.Block;
 
-final class OpenTelemetryExecInterceptor implements ExecInterceptor {
+public final class OpenTelemetryExecInterceptor implements ExecInterceptor {
```

Comment:
If you need to make this class `public` you should move it to `internal` package to avoid it being a part of the public api.